website/content/en/preview/faq.md

```diff
@@ -83,6 +83,17 @@ Yes, see [Example Provisioner Resource]({{< ref "./provisioner/#example-provisio
 * Attribute-based requests are currently not possible.
 * You can select instances with special hardware, such as gpu.
 
+### How does Karpenter "know" which instance type to use?
+
+Karpenter batches pending pods and then binpacks them based on CPU, memory, and GPUs required, taking into account node overhead, VPC CNI resources required, and daemon sets that will be packed when bringing up a new node.
+By default Karpenter uses all available instance types, but it can be constrained in the provisioner spec with the instance types well known label in the requirements section.
```

Comment:
`but it can be constrained in the provisioner spec with the instance types well known label in the requirements section` I think we're missing something in this statement?

Reply:
Would it be better to say this?: ...but it can be constrained in the provisioner spec by adding the well-known label of each instance you want to allow to the requirements section.